developer/extension-point.rst

```diff
@@ -0,0 +1,85 @@
+Extension points
+================
+
+The following describes some of the extension points and how they are used.
+
+VerbExtensionPoint
```

Comment:
Worthwhile to link these to the actual code in question, eg with `:class:`?

Reply:
Unfortunately we don't have generated API docs at the moment. Once that is the case it would be great to link to it.

Reply:
Oh, ha. That's right, I'm forgetting how Sphinx actually works...